### PR TITLE
Fix/stupid typos

### PR DESCRIPTION
**Pull request by martin-cs**
- [ ] Each commit message has a non-empty body, explaining why the change was made.
- [ ] Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- [ ] The feature or user visible behaviour I have added or modified has been documented in the User Guide in doc/cprover-manual/
- [ ] Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- [ ] My commit message includes data points confirming performance improvements (if claimed).
- [ ] My PR is restricted to a single feature or bugfix.
- [ ] White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

File: src/config.inc [hunk 1]
```diff
@@ -10,10 +10,10 @@ endif
 
 # Select optimisation or debug info
 #CXXFLAGS += -O2 -DNDEBUG
-#CXXFLAGS += -O0 -g
+CXXFLAGS += -O0 -g
```
Comment (tautschnig) [inline src/config.inc]: Trying to sneak in a change? :-P

File: src/config.inc [hunk 2]
```diff
@@ -10,10 +10,10 @@ endif
...
 
 # With GCC this adds function names in stack backtraces
-#LINKFLAGS="-rdynamic"
+#LINKFLAGS=-rdynamic
```
Comment (tautschnig) [inline src/config.inc]: If we're doing this, can we also add spaces around the `=`?

**Review by tautschnig — CHANGES_REQUESTED** [review]

File: src/config.inc (hunk 1 above)
Comment (chrisr-diffblue) [reply src/config.inc]: @tautschnig got there before me :-P

File: src/config.inc (hunk 2 above)
Comment (chrisr-diffblue) [reply src/config.inc]: +1

File: src/config.inc (hunk 1 above)
Comment (martin-cs) [reply src/config.inc]: Shit.  Sorry.

File: src/config.inc (hunk 2 above)
Comment (martin-cs) [reply src/config.inc]: Sure.

**Comment by martin-cs** [comment]
@tautschnig and @chrisr-diffblue (who are both actually awake on a Friday morning; unlike some of us) : fixed and should be good to go.

**Review by chrisr-diffblue — APPROVED** [review]

**Review by tautschnig — APPROVED** [review]

**Review by allredj — COMMENTED** [review]
:warning:
This PR failed Diffblue compatibility checks (cbmc commit: 9004dfc96aa978a3a7d61bf919a933d616bc2770).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/105422817
Status will be re-evaluated on next push.
Common spurious failures:
  - the cbmc commit has disappeared in the mean time (e.g. in a force-push)
  - the author is not in the list of contributors (e.g. first-time contributors).

  - the compatibility was already broken by an earlier merge.

**Comment by martin-cs** [comment]
@allredj : I can't access the URL that gives the reason for this.  Given the PR I think it is spurious; please can you have a look?

**Pull request merged**